cpp/src/strings/replace/replace_nulls.cu

```diff
@@ -0,0 +1,82 @@
+/*
```

Comment:
Code moved from https://github.com/rapidsai/cudf/blob/3ba63c3c3cb72950adc4c9699fcfa1a72796a041/cpp/src/strings/replace/replace.cu#L709-L748
(and modified to use offsetalator)